app/views/pages/home.html.haml

```diff
@@ -39,14 +39,60 @@
 #home-map-jumbotron
 .main-section.sponsors
   %h1 Sponsors
-  .row.logos
-    %ul.small-block-grid-1.medium-block-grid-2
-      %li= image_tag 'sponsors/codehs.png', alt: 'CodeHS'
-      %li= image_tag 'sponsors/hackreactor.png', alt: 'Hack Reactor'
-      %li= image_tag 'sponsors/workflowy.png', alt: 'Workflowy'
-      %li= image_tag 'sponsors/makeschool.svg', alt: 'Make School'
-      %li= image_tag 'sponsors/nitrousio.svg', alt: 'Nitrous.IO'
-      %li= image_tag 'sponsors/rackspace.svg', alt: 'Rackspace'
+  .show-for-small-only
+    .row.logos
+      .small-6.columns
```

Comment:
It would be much more elegant to use a grid system for all sizes, but it loses some of the finer control (ie. changing the logo order based on screensize).